src/sensor_state_data/sensor/device_class.py

```diff
@@ -96,6 +96,9 @@ class SensorDeviceClass(BaseDeviceClass):
     # signal strength (dB/dBm)
     SIGNAL_STRENGTH = "signal_strength"
 
+    # specific gravity
+    SPECIFIC_GRAVITY = "specific_gravity"
```

Comment:
https://en.m.wikipedia.org/wiki/Relative_density

Apparently specific gravity is deprecated ? At least according to Wikipedia

Reply:
It may not be the accepted terminology among the scientific community anymore, but it is almost exclusively the terminology used among the beer brewing community, and this is a sensor for homebrewers. 

It does feel pretty niche to add to a library, but it didn't seem like there was a way around that.